tests/phpunit/integration/Core/Storage/Google/OptionsTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * OptionsTest
+ *
+ * @package   Google
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Storage;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Tests\TestCase;
+
+/**
+ * @group Storage
+ */
+class OptionsTest extends TestCase {
+
+	public function test_get() {
+		$options = new Options( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+		delete_option( 'test_option' );
+
+		$this->assertFalse( $options->get( 'test_option' ) );
+		update_option( 'test_option', 'test-value' );
+
+		$this->assertEquals( 'test-value', $options->get( 'test_option' ) );
+	}
+
+	/**
+	 * @group ms-required
+	 */
+	public function test_network_mode_get() {
```

Comment:
Tests annotated like that should also start with a `if ( ! is_multisite() )` condition that skips the respective test.

Reply:
This is handled automatically by `\WP_UnitTestCase_Base::checkRequirements`.

See https://github.com/WordPress/wordpress-develop/blob/ed1233d05f0ea12df0683dcbf38734aa69e39269/tests/phpunit/includes/abstract-testcase.php#L773-L781

Reply:
These groups should be skipped automatically by [`\WP_UnitTestCase_Base::checkRequirements`](https://github.com/WordPress/wordpress-develop/blob/ed1233d05f0ea12df0683dcbf38734aa69e39269/tests/phpunit/includes/abstract-testcase.php#L773-L781). It worked when I tested in multisite mode locally.

I'll take a closer look and see what's going on there.

Reply:
The latest fix is great, polyfilling 5.1 - awesome!